### PR TITLE
support bulk insert in cursor's executemany() method

### PR DESCRIPTION
**Pull request by alexkorotkikh**

### CODE REVIEW

File: tests/test_dbapi.py [hunk 1]
```diff
@@ -117,17 +117,16 @@ def test_executemany(self, festung, cursor):
...
         cursor.executemany(query, params_list)
-        for params, data in zip(params_list, festung.json_requests):
-            assert data['sql'] == query
-            assert data['params'] == params
-        assert cursor.rowcount == sum(n_rows_changed)
+
+        requests = festung.json_requests
+        assert len(requests) == 1
+        assert requests[0]['sql'] == query
+        assert requests[0]['params'] == params_list
```
Comment (mdomke) [inline tests/test_dbapi.py]: This should be `'bulk_params'`!

File: festung/dbapi.py [hunk 2]
```diff
@@ -161,13 +161,9 @@ def execute(self, operation, parameters=None):
...
 
     def executemany(self, operation, parameters_sequence):
-        rowcount = 0
-        for parameters in parameters_sequence:
-            self.execute(operation, parameters)
-            rowcount += self._rowcount
-            if self.fetchone() is not NO_MORE_ROW:
-                raise ProgrammingError("The statement shall not produce a result.")
-        self._rowcount = rowcount
+        self.execute(operation, parameters_sequence)
```
Comment (mdomke) [inline festung/dbapi.py]: If you use `execute` it will use `params` in the request instead of `bulk_params`.

**Review by mdomke — CHANGES_REQUESTED** [review]

**Review by lad1337 — CHANGES_REQUESTED** [review]
```
def execute(self, operation, parameters=None):
#should be something like
def execute(self, operation, parameters=None, bulk_parameters=None):
```

File: festung/dbapi.py (hunk 2 above)
Comment (alexkorotkikh) [reply festung/dbapi.py]: lol, stupid me.... thanks!

**Review by mdomke — APPROVED** [review]

**Pull request merged**